tls/s2n_resume.h

```diff
@@ -75,7 +75,7 @@ int s2n_encrypt_session_ticket(struct s2n_connection *conn, struct s2n_stuffer *
 int s2n_decrypt_session_ticket(struct s2n_connection *conn, struct s2n_stuffer *from);
 int s2n_encrypt_session_cache(struct s2n_connection *conn, struct s2n_stuffer *to);
 int s2n_decrypt_session_cache(struct s2n_connection *conn, struct s2n_stuffer *from);
-int s2n_config_is_encrypt_decrypt_key_available(struct s2n_config *config);
+S2N_RESULT s2n_config_is_encrypt_decrypt_key_available(struct s2n_config *config, bool *out);
```

Comment:
Why do we call this the encrypt_decrypt key instead of just the encrypt key :/

Reply:
I believe it's because the function looks for a key that can both encrypt and decrypt.

Reply:
Sure, but while we have keys that can decrypt but not encrypt anymore, we don't have keys that can encrypt but not decrypt. That would be kind of silly. So encrypt implies decrypt.

Reply:
I see. I can change this to s2n_config_is_encrypt_key_available() instead